packages/@dcl/inspector/src/lib/data-layer/host/undo-redo.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { EntityType } from '@dcl/schemas'
 import * as components from '@dcl/ecs/dist/components'
 
-import { initTestEngine } from './utils'
+import { initTestEngine } from '../../../../test/data-layer/utils'
```

Comment:
if we are moving all the specs to the current folder (why are we doing this ? ) this file should also be here and not isolated in a test folder.
Also with this approach we cant do integration tests. All the test always match a file. 🤔

Reply:
integration test could actually stay in the `test` folder.
I'm doing this movement as I had understood we were going to apply [`dApps testing standards`](https://www.notion.so/decentraland/Testing-standards-46797744fccf4f3eba52335f9866d0eb)